lib/domain/model/contact/contacts_manager_agruments.dart

```diff
@@ -0,0 +1,20 @@
+import 'package:equatable/equatable.dart';
+import 'package:fluffychat/domain/contact_manager/contacts_manager.dart';
+import 'package:fluffychat/domain/model/contact/contact.dart';
+
+class ContactsManagerArguments extends Equatable {
```

Comment:
Create an abstraction first, in the future a lot of kinds of things need to add to Contact Manager